runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/DecompilationController.java

```diff
@@ -320,19 +322,46 @@ private void loadClassBytecode(String name) {
         bytecodeDecompilerView.reloadTextField(name, decompiledClass, bytes);
     }
 
+    public String getVm() {
+        if (vmInfo == null) {
+            return null;
+        }
+        return vmInfo.getVmId();
+    }
+
+    public String[] getOverrides() {
+        AgentRequestAction request = createRequest(vmInfo, AgentRequestAction.RequestAction.OVERRIDES, "");
+        String response = submitRequest(request);
+        if ("ok".equals(response)) {
+            VmDecompilerStatus vmStatus = vmInfo.getVmDecompilerStatus();
+            String[] classes = vmStatus.getLoadedClassNames();
+            return classes;
+        } else {
+            throw new RuntimeException(response);
+        }
+    }
+
+    public void removeOverrides(String pattern) {
+        AgentRequestAction request = createRequest(vmInfo, RequestAction.REMOVE_OVERRIDES, pattern);
+        String response = submitRequest(request);
+        if (!"ok".equals(response)) {
+            throw new RuntimeException(response);
+        }
+    }
+
     class QuickCompiler {
 
         public void upload(String clazz, byte[] body) {
             CommonUtils.uploadByGui(vmInfo, vmManager, new CommonUtils.StatusKeeper() {
                 @Override
                 public void setText(String s) {
-                    GlobalConsole.getConsole().addMessage(Level.ALL, s);
+                    GlobalConsole.getConsole().addMessage(Level.WARNING, s);
```

Comment:
Because that message is crucial, not debug only. 
I think I messed up Levels when I was writting that  - I thought I.m using jrd.Level but instead there is log.Level. And those Level.ALL have exactly oposite meaninf in jrd.Level and logLevel. Seeing your reply, I think You messed too ;)